### PR TITLE
Updated Readme.md to show valuable contributors

### LINKED ISSUE #81: Want to add Contributors section into main README.md
I think it looks cool and it will automatically updated whenever a contributor contribute to this project.

### PR DESCRIPTION
**Pull request by malay5**
## Description

Added Contributors section, showing all the contributors, who have contributed to the project

## Related Issue(s)

Fixes #81

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,3 +74,10 @@ Know how to code and you'd like to add a feature? Feel free to [create a PR](htt
 ## Bug Reports
 Spot an issue or bug? Please let us know by [creating a bug report](https://github.com/alec-kr/F1PyStats/issues/new/choose) :bug:
 
+## OUR VALUABLE CONTRIBUTORS✨
```
Comment (alec-kr) [inline README.md:77]: Do you want this heading to be in all capital letters? The rest of the document headings have each first letter capitalized.

**Comment by alec-kr** [comment]
Feature seems great. Thank you for contributing 😄

**Review by alec-kr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (malay5) [reply README.md:77]: I will change it to the requested font

**Comment by malay5** [comment]
Please check the updated PR

**Comment by malay5** [comment]
Could you please add this PR under hacktoberfest22?

**Review by alec-kr — APPROVED** [review]
Excellent!

I will merge it now.

**Pull request merged**